packages/js/client/src/plugin/PluginWeb3Api.ts

```diff
@@ -88,7 +88,13 @@ export class PluginWeb3Api extends Api {
           )) as Record<string, unknown>;
         }
 
-        this._getInstance().loadEnv(env, module);
+        const plugin = this._getInstance();
+        if (plugin && plugin.loadEnvByModule) {
+          plugin.loadEnvByModule(module, env);
+        } else {
+          // Deprecated function
```

Comment:
We should create an issue to remove this once all plugins are migrated

Reply:
@dOrgJelli already created one for migrating plugins and we will be removing this in the same PR.